File: en/philosophy.js
(function philosophy() {
    var question = 'What is ';
    var idea     = 'philosophy';
    var prefix   = 'philosophy of ';
    var symbol   = '?';

    var iteration = '';

    // assume 42 is infinite ∞
    for (var i = 0; i < 42; i++) {
        var output = question + iteration + idea + symbol;

        console.log(output);

        iteration = prefix + iteration;
    }
})();

File: en/philosophy.c
#include <stdio.h>
#include <string.h>

int main () {
    char question[] = "What is ";
    char idea[]     = "philosophy";
    char prefix[]   = "philosophy of ";
    char symbol[]   = "?";

    char iteration[42] = ""; // assume 42 is infinite ∞

    while(1) {
        printf("%s%s%s%s\n", question, iteration, idea, symbol);
        strcat(iteration, prefix); // a logical bug it is
    }
}
